automation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:15.04
+
+# Update index
+RUN apt-get update && apt-get -y upgrade
+
+# Install the following utilities (required by poky)
+RUN apt-get install -y build-essential chrpath curl diffstat gcc-multilib gawk git-core libsdl1.2-dev texinfo unzip wget xterm cpio file
```

Comment:
This needs to be `apt-get update && apt-get install ...` otherwise we may hit a case of trying to build with out of date package lists and having it fail due to that (same applies for all other `apt-get install`s)